v2/commands/job/get.go

```diff
@@ -111,21 +96,33 @@ func Get(cmd *cobra.Command, args []string) error {
 			return config.UsageErrorf("Error setting reverse: %w", err)
 		}
 
+		// --latest N == --limit N --reverse --sort created
+		if latest > 0 {
+			limit = latest
+			reverse = true
+			sort = common.SortCreated
+		}
+
 		var items []oapi.HTCJob
 		params := oapi.GetJobsParams{
 			ProjectId: p.ProjectId,
 			TaskId:    p.TaskId,
 			Group:     oapi.OptString{group, group != ""},
-			PageSize:  oapi.NewOptInt32(pageSize),
+			PageSize:  oapi.NewOptInt32(common.PageSize),
 			ViewType:  oapi.NewOptViewType(oapi.ViewTypeFULL),
 		}
+
+		if filterStatus != "" {
+			params.Status = oapi.NewOptRescaleJobStatus(filterStatus.ToRescaleStatus())
+		}
+
 		for {
 			res, err := getJobs(ctx, runner.Client, &params)
 			if err != nil {
 				return err
 			}
 			items = append(items, res.Items...)
-			if limit > 0 && len(items) >= limit {
+			if !(sortOrReversedPassed || latest > 0) && limit > 0 && len(items) >= limit {
```

Comment:
You're fine as is, but it seems slightly cleaner to check `sort` and `reverse` here instead of referring to the intermediate variables `latest` and `sortOrReversedPassed`. (Those only needed to exist during flag parsing; if this part of the code were a separate function, for instance, it would need no knowledge of what the original flags were - just `limit`, `sort`, and `reverse`.)

```suggestion
			// Only stop consuming pages when no sort/reverse is in effect.
			if sort == "" && ! reverse && limit > 0 && len(items) >= limit {
```

Reply:
I believe you would still need `&& ! latest > 0` in the condition. So, the full one will be:
```
if sort == "" && ! reverse && ! latest > 0 && limit > 0 && len(items) >= limit {
```

You would need it because as we discussed latest is [shorthand](https://github.com/rescale-labs/htc-cli/pull/9/files#diff-4c6da3c611b0ccd9c423616028b1e81855688b4f9e9c344dc94118a509c5270cR100-R104) for limit with sort and reverse:
```
if latest > 0 {
  limit = latest
  reverse = true
  sort = common.SortCreated
}
```

I prefer keeping code as is.

Also, one thing I find a bit confusing working with cobra is setting the defaults for options user haven't passed in. So, I made a case here for using the flags.Changed("flagName") instead of checking defaults.